internetarchive/openlibrary
# Fixes Remove `focusin` and `focusout` polyfill from vendor/ jquery-validate

Closes #2419

### Technical
Updated `jquery-validate`'s version from `1.6` to `1.19.1`.

### Testing
1. Open developer console.
2. Check the console to make sure that the `jquery-migrate.js:41 JQMIGRATE: jQuery.browser is deprecated` is gone for the `jquery.validate.js` file.
3. Login as admin
4. Click on add a book.
5. Try to add a book without any of the fields filled out.  See that the required fields turn red still and prevent you from continuing.
6. Fill in those fields in now and when on the next screen delete the title and try to submit the form.  You should see the required Title field turn red and prevent you from continuing.

### Evidence
Add a book form:
![required1](https://user-images.githubusercontent.com/7445503/66262765-307b6200-e7b5-11e9-8665-2402d24a2555.png)

Add more information form:
![required2](https://user-images.githubusercontent.com/7445503/66262779-5143b780-e7b5-11e9-83b2-dec139fe6790.png)

Developer Console:
![dev_console](https://user-images.githubusercontent.com/7445503/66262804-9536bc80-e7b5-11e9-9a69-a20093c15ba6.png)
